clustergroup/templates/core/catalog-sources.yaml

```diff
@@ -1,13 +1,14 @@
 {{- if not (eq .Values.enabled "plumbing") }}
 {{- range .Values.clusterGroup.indexImages }}
+{{- $name := mustRegexReplaceAll "[^/]*/(.*):.*" .image "${1}" | replace "/" "-" }}
 apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
-  name: {{ .name }}
+  name: {{ coalesce .name $name }}
   namespace: openshift-marketplace
 spec:
   sourceType: grpc
-  image: {{ .image }}:{{ .version }}
```

Comment:
Is this change here intentional? Are we dropping support for .version? Should we drop it from test.yaml as well?

Reply:
the image we get from the UMB message includes the version